### ISSUE TITLE
Google Earth won't load unless ?beta=1 is passed

### ISSUE THREAD

**Issue by UndarkAido** [jiahaog/nativefier#846]
### Description

I'm trying to create an app that loads `https://earth.google.com/web/`, but it always hangs at the loading screen.

### Steps to reproduce issue

Create a Nativefier app pointing to `https://earth.google.com/web/` I used this JSON:
```
"options": {
	"name": "GoogleEarth",
	"targetUrl": "https://earth.google.com/web/",
	"overwrite": true,
	"out": "temp",
	"icon": "resources/IconManager/output/Internet/Google/GoogleEarth-Icon[Square].png",
	"browserwindowOptions": "{\"webPreferences\": {\"webviewTag\": true }}",
	"enableEs3Apis": true
}
```

### Details

- Are you nativefying a *public* website? Yes
- Feature request? Have you looked at `nativefier --help` to see if an existing option could fit your needs? No?
- Full nativefier command used to build your app: See above, built using Aidoboy/nativefier-install
- Version of Nativefier (run `nativefier --version`): `v7.7.0`
- Version of node.js (run `node --version`): `v12.9.1`
- OS: `Windows 10`
- Error message / stack trace (if any):

```
app.js:3247 [Deprecation] 'window.webkitStorageInfo' is deprecated. Please use 'navigator.webkitTemporaryStorage' or 'navigator.webkitPersistentStorage' instead.
IV @ app.js:3247
undefined:1 Failed to load resource: the server responded with a status of 404 ()
DevTools failed to parse SourceMap: https://earth.google.com/web/preload.js.map

```

**Comment by mxdpeep** [comment]
have you tried different electron versions? the most current is 6.0.7
I managed to build it and it says: "Unfortunately your computer does not support WebGL graphics acceleration; Google Earth cannot be loaded. Please try another device."

**Comment by Aidoboy** [comment]
I have not tried different Electron versions yet, but I did find [https://earth.google.com/web?beta=1](https://earth.google.com/web?beta=1), which does load.

**Comment by Aidoboy** [comment]
You can see the GPU-munching results here: https://steamcommunity.com/sharedfiles/filedetails/?id=1852887871

**Comment by mxdpeep** [comment]
> You can see the GPU-munching results here: https://steamcommunity.com/sharedfiles/filedetails/?id=1852887871

maybe some common resolutions? like 1920x1080 or similar?

**Comment by mxdpeep** [comment]
> I have not tried different Electron versions yet, but I did find https://earth.google.com/web?beta=1, which does load.

does not work for me, same GPU bug

**Comment by Aidoboy** [comment]
Here's a 1080p version:
https://steamcommunity.com/sharedfiles/filedetails/?id=1852891599

**Comment by ronjouch** [comment]
This appears fixed in 8.0.4; closing. Please shout back with fresh debug info if I'm missing anything.
